using System;

namespace Xero.RefactoringExercise.Domain.Exceptions
{
    public class RecordNotFoundException : Exception
    {
    }
}
